### PR TITLE
refactor: Update rxjs operators to be lettable.

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

**Comment by Brocco** [comment]
Dependent upon angular/angular-cli#8976

File: packages/angular_devkit/core/node/cli-logger.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import { logging, terminal } from '@angular-devkit/core';
-import 'rxjs/add/operator/filter';
+import { filter } from 'rxjs/operators';
```
Comment (CaerusKaru) [inline packages/angular_devkit/core/node/cli-logger.ts]: This should be an import directly from `'rxjs/operators/filter'`

File: packages/angular_devkit/core/src/logger/transform-logger_spec.ts [hunk 2]
```diff
@@ -6,21 +6,19 @@
  * found in the LICENSE file at https://angular.io/license
  */
 // tslint:disable:no-any
-import 'rxjs/add/operator/filter';
-import 'rxjs/add/operator/map';
+import { filter, map, toArray } from 'rxjs/operators';
```
Comment (CaerusKaru) [inline packages/angular_devkit/core/src/logger/transform-logger_spec.ts]: These should be separate imports:

import { filter } from 'rxjs/operators/filter';
...

Comment (CaerusKaru) [reply packages/angular_devkit/core/src/logger/transform-logger_spec.ts]: Per discussion in https://github.com/angular/angular-cli/pull/8976#discussion_r158592854, this is a non-starter/not necessary

**Pull request merged**